agent/ansible/pbench/agent/roles/pbench_agent_install/README.md

```diff
@@ -0,0 +1,43 @@
+pbench_agent_install
+==========================
+
+This role is used to install the latest versions of the pbench-agent and pbench-sysstat RPMs.
+
+Requirements
+------------
+
+Role Variables
+--------------
+
+Dependencies
+------------
+This role depends on another role in this collection:
+
+- pbench_clean_yum_cache
+
+Example Playbook
```

Comment:
Can we reference an example playbook in the git repo?

Reply:
See above.

Reply:
Done